naucse/models.py

```diff
@@ -71,7 +72,22 @@ def latex(self):
 
     @reify
     def css(self):
-        return self.info.get('css')
+        """
+        If the lesson defines extra css, the scope of the styles is limited to ``.lesson-content``,
+        a div which contains the actual lesson content. That way the styles can't disrupt
+        overall page appearance.
```

Comment:
> That way the styles can't disrupt overall page appearance.

That is wrong. You can't rely on this function with untrusted input.
For example, this CSS would break pages:

```css
    .dummy, * {
        position: absolute;
    }
```